js/my.js

```diff
@@ -232,3 +232,24 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  var args = cmd.split(' ');
+
+  if (args && args[0] === '/query' && args[1]) {
+    var to = args[1];
+    var msg = privMSG[to];
+
+    if (!msg) {
+      msg = privMSG[to] = new Tab({
```

Comment:
there's no need to assign to the variable `msg` here, since we don't use it after assignment.  We can get rid of it:
```coffeescript
if !privMSG[to]
  privMSG[to] = ...
```